backend/fms_core/services/library.py

```diff
@@ -48,3 +51,96 @@ def create_library(library_type, index, platform, strandedness, library_size=Non
         errors.append(';'.join(e.messages))
 
     return library, errors, warnings
+
+
+def convert_library(process, platform, sample_source, concentration, library_size, container_destination,
```

Comment:
Is there any reason to pass the library_size and concentration here ? Even if we do not reset them on conversion, we can still get them from the sample_source.